### PR TITLE
web: update step3 and add account

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are updating those 2 screens on web-frontend.

### CODE REVIEW

File: web-frontend/src/i18n/en_US.json [hunk 1]
```diff
@@ -18,11 +18,14 @@
...
       "step3": {
-        "title": "Backup"
+        "title": "Backup",
+        "description": "Security is very important, we recommend backing up your account using a key pair or a .yaml file. Would you like to do it now or later?"
       },
       "buttons": {
         "define_password": "Define a password",
-        "next": "Next"
+        "next": "Next",
+        "backup": "Back up",
```
Comment (mazmassa) [inline web-frontend/src/i18n/en_US.json:27]:
```suggestion
        "backup": "Backup",
```

Comment (maxime-carabina) [reply web-frontend/src/i18n/en_US.json:27]: One the design, it's "back up".
So what would I do ?

Comment (mazmassa) [reply web-frontend/src/i18n/en_US.json:27]: Oh! Check with him then!

**Review by peterjah — APPROVED** [review]

File: web-frontend/src/i18n/en_US.json (hunk 1 above)
Comment (Thykof) [reply web-frontend/src/i18n/en_US.json:27]: > Backup is a noun spelled as a single word, while back up is a verb spelled as 2 words

Comment (Thykof) [reply web-frontend/src/i18n/en_US.json:27]: here either you skip, or you back up (2 verbs)

**Pull request merged**